src/pages/careers.tsx

```diff
@@ -0,0 +1,287 @@
+        super(props);
+        this.state = {
+            isModalOpen: false
+        }
+        this.openModal = this.openModal.bind(this);
+    }
+
+    openModal() {
+        this.setState({ isModalOpen: true })
+    }
+
+    render() {
+        return <IndexLayout title="Gitpod - Careers" canonical="https://www.gitpod.io/careers">
+            <Page>
+                <Container>
+                    <GitGraph left={-900} top={40} graph={[
+                        { start: [0, 115] },
+                        { right: 860 },
+                        {},
+                        { right: 10 },
+                        { down: 200 },
+                        {
+                            fork: [
+                                { right: 115 },
+                                {
+                                    onClick: () => { }
+                                },
+                                { right: 163 },
+                                {
+                                    onClick: () => { }
+                                },
+                                { right: 163 },
+                                {
+                                    onClick: () => { }
+                                },
+                                { right: 163 },
+                                {
+                                    onClick: () => { }
+                                },
+                                { right: 163 },
+                                {
+                                    onClick: () => { }
+                                },
+                                { right: 163 },
+                                {
+                                    onClick: () => { }
+                                }
+                            ]
+                        },
+                        { down: 205 },
+                        {},
+                        { down: 600 },
+                        {},
+                        { down: 1725 },
+                        { right: 400 },
+                        {}
+                    ]} />
+                    <Logos logos={[
+                        [1050, 125, 40],
+                    ]} />
+                    <div style={{ marginTop: 60 }}>
+                        <h4 style={{ color: colors.fontColor2 }}>We're Hiring</h4>
+                        <h1>Become Part of the Team</h1>
+                        <div style={{ height: 3, width: 95, backgroundColor: colors.brand, marginBottom: 30 }} />
+                        <div style={{ display: 'flex', flexDirection: 'row', justifyContent: 'space-between', marginTop: 50 }}>
+                        </div>
+                    </div>
+                    <p>
+                        Our Employees are the core of Gitpod and sharing its future.
+                        <br />
+                        We're convinced that great results are built on culture of trust, empathy and productivity. That's why working with us means
```

Comment:
Please add a `:` at the end:

```suggestion
                        We're convinced that great results are built on culture of trust, empathy and productivity. That's why working with us means:
```